### PR TITLE
Add Yaml config for some op

### PR DESCRIPTION
**Pull request by zyfncg**
### PR types

Others

### PR changes

OPs

### Describe

Pcard-67742

为`nanmedian`和`stanh`算子增加Yaml定义配置。

### CODE REVIEW

**Review by heavyrain-lzy — APPROVED** [review]

File: paddle/phi/api/yaml/fused_backward.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # This file is designed for fusion C++ backward operators, which manages the
-# generated code for dynamic mode and static mode.
-# The operators in the file have extra configuration item "support_dygraph_mode".
-# If one operator have "support_dygraph_mode : True", it supports dygraph mode.
+# generated code for static mode and dynamic mode (when `support_dygraph_mode` is true).
+# "support_dygraph_mode" is and extra configuration item in this file,
```
Comment (jiahy0825) [inline paddle/phi/api/yaml/fused_backward.yaml:3]: `and -> an`

File: paddle/phi/api/yaml/fused_ops.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # This file is designed for fusion C++ farward operators, which manages the
-# generated code for dynamic mode and static mode.
-# The operators in the file have extra configuration item "support_dygraph_mode".
-# If one operator have "support_dygraph_mode : True", it supports dygraph mode.
+# generated code for static mode and dynamic mode (when `support_dygraph_mode` is true).
+# "support_dygraph_mode" is and extra configuration item in this file,
```
Comment (jiahy0825) [inline paddle/phi/api/yaml/fused_ops.yaml:3]: `and -> an`

**Review by jiahy0825 — APPROVED** [review]
LGTM except for two typo errors.

File: paddle/phi/api/yaml/fused_backward.yaml (hunk 1 above)
Comment (zyfncg) [reply paddle/phi/api/yaml/fused_backward.yaml:3]: Ok, fix  it in next pr.

File: paddle/phi/api/yaml/fused_ops.yaml (hunk 2 above)
Comment (zyfncg) [reply paddle/phi/api/yaml/fused_ops.yaml:3]: Ok, fix  it in next pr.

**Pull request merged**